ui/src/index.js

```diff
@@ -74,12 +74,19 @@ window.addEventListener('keyup', event => {
 const history = syncHistoryWithStore(browserHistory, store)
 
 const Root = React.createClass({
+  getInitialState() {
```

Comment:
is there a reason you all didn't use extends component ... init state at connstructor?